backend/routes/paypal.js

```diff
@@ -227,15 +234,75 @@ module.exports = function (router) {
 
       log.debug(JSON.stringify(event, null, 4))
 
-      if (event.event_type !== 'PAYMENT.CAPTURE.COMPLETED') {
-        log.debug(`Ignoring event ${event.type}`)
+      const eventType = get(event, 'event_type')
+
+      if (
+        !['PAYMENT.CAPTURE.PENDING', 'PAYMENT.CAPTURE.COMPLETED'].includes(
+          eventType
+        )
+      ) {
+        log.debug(`Ignoring event ${eventType}`)
         return res.sendStatus(200)
       }
 
+      const isCompleted = eventType === 'PAYMENT.CAPTURE.COMPLETED'
+
+      if (isCompleted) {
+        // Check if it is an existing pending order
+
+        const order = await Order.findOne({
```

Comment:
Moving forward, I've been trying to keep the routes skinny and put the logic under logic/
So it would probably make sense to move this logic code to logic/order/order.js in a new "processPendingOrder" method.